### PR TITLE
docs(connector-kit):icon format changed to svg

### PR DESCRIPTION
**Pull request by Unsharm**
## Description
In this pull request, I have changed the format of the icon's image in the adoption view of the Connector kit from PNG to SVG.

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [X] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [X] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Review by SujitMBRDI — CHANGES_REQUESTED** [review]
Hello @Unsharm,
As i mentioned on other pull request [https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/773#pullrequestreview-1950745914](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/773#pullrequestreview-1950745914), for connector-kit, i can see the banner is being used on changelog page as well and not being used on operation view.
So, kindly adapt the changes on changelog page with icon you have created in svg format.

**Comment by SujitMBRDI** [comment]
Hi @maximilianong,
wanted to confirm, do we need to delete old .png image from static/img folder? if yes then we need to adapt changes to old versions as well because it can loose the reference.

**Review by SujitMBRDI — CHANGES_REQUESTED** [review]

**Comment by maximilianong** [comment]
> Hello @Unsharm, As i mentioned on other pull request [#773 (review)](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/773#pullrequestreview-1950745914), for connector-kit, i can see the banner is being used on changelog page as well and not being used on operation view. So, kindly adapt the changes on changelog page with icon you have created in svg format.

Can we do this for all KITs the same way. Icon on all main pages of a KIT.
I was not aware of this.

**Comment by maximilianong** [comment]
> Hi @maximilianong, wanted to confirm, do we need to delete old .png image from static/img folder? if yes then we need to adapt changes to old versions as well because it can loose the reference.

No - that topic I discussed with @Unsharm aswell. We leave the old files where they are.

**Comment by SujitMBRDI** [comment]
> > Hi @maximilianong, wanted to confirm, do we need to delete old .png image from static/img folder? if yes then we need to adapt changes to old versions as well because it can loose the reference.
> 
> No - that topic I discussed with @Unsharm aswell. We leave the old files where they are.

Cool :)

File: docs-kits/kits/Connector Kit/page_changelog.md [hunk 1]
```diff
@@ -1,11 +1,25 @@
 # Changelog
 
-![Connector kit banner](@site/static/img/ConnectorKitIcon.png)
+![Connector kit banner](/img/kit-icons/connector-kit-icon.svg)
 
 ## Connector KIT
 
 All notable changes to this Kit will be documented in this file.
 
+## [2.2.0] - unreleased
```
[the remark this answers is not in the document]
Comment (Unsharm) [reply docs-kits/kits/Connector Kit/page_changelog.md:9]: > Here, version should be either [3.0.0] or [2.1.0] as in last release 24.03 version [2.0.0] is released.

@SujitMBRDI okay, I am adapting the change.

**Comment by SujitMBRDI** [comment]
> > Hello @Unsharm, As i mentioned on other pull request [#773 (review)](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/773#pullrequestreview-1950745914), for connector-kit, i can see the banner is being used on changelog page as well and not being used on operation view. So, kindly adapt the changes on changelog page with icon you have created in svg format.
> 
> Can we do this for all KITs the same way. Icon on all main pages of a KIT. I was not aware of this.

I thought in a way that for other KITs where ever we see .png icon then we should replace it. 
Lets take this as example, if we don't see banner on operation view then we should ignore it but for those main pages having icon that has to be replace by .svg image
Please correct me if i am wrong here.

**Comment by maximilianong** [comment]
> > > Hello @Unsharm, As i mentioned on other pull request [#773 (review)](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/773#pullrequestreview-1950745914), for connector-kit, i can see the banner is being used on changelog page as well and not being used on operation view. So, kindly adapt the changes on changelog page with icon you have created in svg format.
> > 
> > 
> > Can we do this for all KITs the same way. Icon on all main pages of a KIT. I was not aware of this.
> 
> I thought in a way that for other KITs where ever we see .png icon then we should replace it. Lets take this as example, if we don't see banner on operation view then we should ignore it but for those main pages having icon that has to be replace by .svg image Please correct me if i am wrong here.

TBH would be nice if we see that on operation view is no KIT icon - we add it.

File: docs-kits/kits/Connector Kit/page_changelog.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Unsharm) [reply docs-kits/kits/Connector Kit/page_changelog.md]: @SujitMBRDI changes has been implemented.

**Review by SujitMBRDI — APPROVED** [review]
LGTM

**Review by maximilianong — APPROVED** [review]

**Pull request merged**